### DISCUSSION TITLE
Setting a job fail condition if two variables are different

### DISCUSSION THREAD

**Discussion by iineh** [BretFisher/ama#217] [Q&A]
Hi, 
Is it possible to set a fail condition if two variables are not the same

Background:
I created a Github workflow to verify a GUID entry on the Database tables of a server and client machine.
I retrieved the entry and passed it on to two GITHUB_ENV vars

          echo "TEMP_1=12" | Out-File -FilePath $Env:GITHUB_ENV -Encoding UTF8 -Append
          echo "TEMP_2=12" | Out-File -FilePath $Env:GITHUB_ENV -Encoding UTF8 -Append
    
      - name: Check acc
        if: ${{env.TEMP_1}} != ${{env.TEMP_2}}
        uses: actions/github-script@v3
        with:
         script: |
          core.setFailed('FAILED')
          

the above always fails the job even if the vairables are different

Thanks

**Comment by BretFisher** [comment]
Note, [stackoverflow.com](https://stackoverflow.com) or [github.community](https://github.community/) would likely be better for general troubleshooting. 

An idea would be to try removing the brackets in the if statement:

`if: env.TEMP_1 != env.TEMP_2`
